src/cascadia/WindowsTerminal/CustomWindowMessages.h

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+// Custom window messages
+#define CM_UPDATE_TITLE (WM_USER)
+#define CM_NOTIFY_FROM_TRAY (WM_APP + 1)

```

Comment:
I'm guessing you chose WM_APP because this message is being originated from a different context, but WM_USER is for internal-to-the-app messages?

You can probably put this one in WM_USER too, but it's up to you. If you want to keep it in WM_APP, add a comment explaining why :smile:

Reply:
Aha yeah WM_USER seems a bit more appropriate, I'll go with that